### PR TITLE
fix: rend valus-istio-gateways.yaml miss cpu.targetAverageUtilization

### PR DESCRIPTION
**Pull request by HYmian**

### CODE REVIEW

**Comment by linsun** [comment]
Does this work with the req cpu commented out?

**Comment by rshriram** [comment]
reassigning to all the proponents of ok-to-test.. This PR is stuck on ok-to-test despite issuing that command. Since you folks were very passionate about ok-to-test, please carry this PR along with whatever set of commands necessary.

**Comment by linsun** [comment]
pls make sure this is answered, i'm not convinced this works without cpu req.

https://github.com/istio/istio/pull/8896#issuecomment-423547260

**Comment by HYmian** [comment]
@linsun
It does not work with cpu req commented out

File: install/kubernetes/helm/istio/values-istio-gateways.yaml [hunk 1]
```diff
@@ -51,6 +51,8 @@ gateways:
       #requests:
       #  cpu: 1800m
       #  memory: 256Mi
+    cpu:
```
Comment (gyliu513) [inline install/kubernetes/helm/istio/values-istio-gateways.yaml]: Can you please add some comments here to clarify that cpu request needs to be set if want the HPA works, would be great to link Kubernetes HPA document here.

Comment (HYmian) [reply install/kubernetes/helm/istio/values-istio-gateways.yaml]: It's done

**Comment by gyliu513** [comment]
@HYmian can you please rebase?

**Pull request closed without merging**